docs/moderator-handbook.md

```diff
@@ -187,11 +194,15 @@ In exceptional circumstances, it may be appropriate for a moderator to fork a di
 
 Again, if you have any problems or questions, make a post with your actions in the `"Staff"` category, and tag another moderator if you want them to review your moderating actions.
 
+### Dealing with Posted Solutions
+
+When a user posts a solution to any of the questions, remove it and either, kindly let them know that solutions are not permitted on the forum in your own words or, use the **Solution Instead of Help** "Canned reply".
+
 ### Underage Users
 
 Our [Terms of Service](https://freecodecamp.org/terms) require that freeCodeCamp users be at least 13 years of age. If a user reveals that they are under the age of 13, send them the below message and delete their forum account (if deletion is not available, suspending the account is sufficient).
 
-**Email `support[at]freecodecamp.org` to delete the user's freeCodeCamp account as well.**
+**Email `support[at]freecodecamp.org` to delete the user's freeCodeCamp account as well (providing the email address and username of the offending forum account).**
```

Comment:
might be worth noting in this section that only admins can access the email addresses of forum users

Reply:
Honestly, I think this flow should be updated entirely.

A moderator should suspend the forum account, send the following message, and then ping a member of staff to delete the accounts - this way we can grab the email address to check for a `/learn` account.

Reply:
Hi Naomi, I think the guide basically says the same thing (but in a slightly different order)?